checker/bin/wpi.sh

```diff
@@ -0,0 +1,170 @@
+#!/bin/bash
+
+# This script performs whole-program inference on a project directory.
+
+# For usage and requirements, see the "Whole-program inference"
+# section of the Checker Framework manual:
+# https://checkerframework.org/manual/#whole-program-inference
+
+
+while getopts "d:u:t:" opt; do
+  case $opt in
+    d) DIR="$OPTARG"
```

Comment:
Is the `-d` command-line argument to `wpi.sh` needed?
Here are two ideas, either of which may simplify invocation of the script:
 * Make it optional, using the current directory if not supplied.
 * Remove it, and take the directory as the first non-command-line argument.  Then there is no need for the `--` on the wpi.sh command line, either.